ccpp/suites/suite_FV3_GFS_v17_cpldnsstras.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v17_cpldnsstras" version="1">
```

Comment:
Moorthi, It looks to me we don't have GFS v17 configuration yet, it may cause some confusion.

Reply:
It is just a SDF that I use - if the management want to restrict, management is welcome to delete them all.

Reply:
Would it be better to rename all these `GFS_v17` to `GFS_v17alpha` to indicate that the configuration is not yet finalized?

Reply:
Since the code will be committed to develop branch and used by broad community, @yangfanglin can we get physics group to give an official name for those suites files? I am worried that developers will be surprised/confused with those suite files when they come into the official repository.